jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarProcessor.java

```diff
@@ -49,4 +65,86 @@ public static JarType determineJarType(Path jarPath) throws IOException {
     }
     return JarType.STANDARD;
   }
+
+  /**
+   * Explode jar and create three layers for classes, resources and dependencies on container.
```

Comment:
```suggestion
   * Explodes jar and create three layers for classes, resources and dependencies on container.
```
and we don't put a period for `@param`, `@return` and such, but honestly, I didn't care.